taccsite_cms/_settings/auth.py

```diff
@@ -14,9 +14,9 @@
 # 1. Set 'LDAP_ENABLED = False'.
 # 2. Duplicate the 'AUTHENTICATION_BACKENDS' setting.
 # 3. Remove its '...LDAPBackend' entry.
-# 4. If 'INCLUDES_CORE_PORTAL = False', remove '...CorePortalAuthBackend' entry.
+# 4. If 'PORTAL_IS_CORE = False', remove '...CorePortalAuthBackend' entry.
```

Comment:
The term `PORTAL_IS_CORE` is ambiguous to me. While some portals have the full `CORE-PORTAL`, some only have the `CORE-CMS`, but ALL portals are `CORE`.

Wouldn't `USES_PORTAL` be clearer as to the relationship being defined here?

Reply:
I do not disagree. I offer facts so we can find the best name.

1. Not all Portals are Core-Portal. TUP uses Core-CMS, but not Core-Portal.
2. In this context, `AUTH_USES_PORTAL` is clearer.
3. There is another context where `PORTAL_IS_CORE` is used: [`/taccsite_cms/urls.py`](https://github.com/TACC/Core-CMS/blob/feat/cmd-79-support-new-core-cms-settings-prefix/taccsite_cms/urls.py#L31-L41).

Perhaps this should become two settings:
- `PORTAL_MANAGES_AUTH` (or `AUTH_USES_PORTAL`, but I am trying to prefix all settings with `PORTAL_`[^1])
- `PORTAL_IS_CORE`

Thoughts?

[^1]: Because that seems to be what other CMS apps do e.g. `djangocms-blog` uses `BLOG_`.
[^2]: So we know at a glance the origin of the setting is a TACC Portal.